### PR TITLE
Update model API to use Foundational Model

### PR DESCRIPTION
**Pull request by bolasim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,8 +162,8 @@ def call_model(request):
     instance_prompt = request_body.get("instance_prompt")
     run_id = request_body.get("run_id")
     run = FinetuningRun(run_id)
-    model = StableDiffusionPipeline(model_id=run.deployed_model.id)
-    image, url = model(instance_prompt)
+    model = run.deployed_model
+    _, url = model(instance_prompt)
```
Comment (amiruci) [inline README.md]: let's keep the `image` even though it's unused. it's informational for the reader.

**Review by amiruci — COMMENTED** [review]
i'll wait for tuhin to approve

**Review by tuhins — APPROVED** [review]

**Comment by amiruci** [comment]
thanks @bolasim!

**Comment by bolasim** [comment]
I don't think this was ready to merge. Waiting on a new baseten_client release

**Pull request merged**